src/main/java/com/autosale/shop/service/AuthenticationService.java

```diff
@@ -12,5 +12,7 @@ public interface AuthenticationService {
 
     boolean isAuthenticated(String token);
```

Comment:
where is method is used?

Reply:
Now it has special endpoint /login/tokens/verify that is used more for testing than for release functionality.

Reply:
removed